mono/metadata/object.c

```diff
@@ -4649,6 +4651,11 @@ create_unhandled_exception_eventargs (MonoObjectHandle exc, MonoError *error)
 {
 	MONO_REQ_GC_UNSAFE_MODE;
 
+	if (0) { // This is at the top instead of the bottom to avoid goto around initialization.
```

Comment:
This is a different solution than the rest.

Reply:
This looks like dead code, rather stay with the extra line than this.

Reply:
It isn't dead because there is a label but I'm ok either way -- and people do not like the other way either..though the other way does work more and maybe, consistency. Let me update then..

Reply:
The old code also had a return before it, so similarly dead, but ok, I'm updating.